PowerShell/JumpCloud Module/Docs/Get-JCCommandResult.md

```diff
@@ -90,88 +90,88 @@ Skips returning the first 100 command results and only returns the 10 results af
 
 ## PARAMETERS
 
-### -ByID
-Use the -ByID parameter when you want to query the contents of a specific Command Result or if the -CommandResultID is being passed over the pipeline to return the full contents of a JumpCloud Command Result.
-The -ByID SwitchParameter will set the ParameterSet to 'ByID' which queries one JumpCloud Command Result at a time.
+### -ByCommandID
+Use the -ByCommandID or -ByWorkflowID parameter when you want to query the results of a specific Command. The -ByCommandID or -ByWorkflowID SwitchParameter will set the ParameterSet to 'ByCommandID' which queries all JumpCloud Command Results for that particular Command.
 
 ```yaml
 Type: System.Management.Automation.SwitchParameter
-Parameter Sets: ByID
+Parameter Sets: ByCommandID
 Aliases:

```

Comment:
I'm not sure why the Aliases are not autopopulated here but I think we can specify them after the fact and they will not be overwritten.